sklearn/impute/_base.py

```diff
@@ -16,10 +16,13 @@
 from ..utils.validation import check_is_fitted
 from ..utils.validation import FLOAT_DTYPES
 from ..utils._mask import _get_mask
+from ..utils import is_pd_na
 from ..utils import is_scalar_nan
 
 
 def _check_inputs_dtype(X, missing_values):
+    if is_pd_na(missing_values):
+        return
```

Comment:
May we leave a comment about why we are skipping the validation here?

Reply:
I skipped the validation so that `pd.NA` can also be used to impute numerical arrays. Added a comment in the code.